keps/sig-api-machinery/2161-apiserver-default-labels/README.md

```diff
@@ -172,14 +139,16 @@ NetworkPolicies or writing admission controller configurations.
 
 * The name of the namespace must satisfy label value [requirements](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set).
 Currently, the name of the namespace must be a valid [DNS label](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names),
-and this requirement must be satisfied going forward.
+and this requirement will NOT CHANGE  going forward.
 
 ### Risks and Mitigations
 
 * Every namespace gets bigger in size. We think this is inconsequential
 * Some user may already be using that label already. This is mitigated by the 
 fact that the whole "kubernetes.io" label prefix is [reserved](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set). However, since there
-is no validation of reserved label prefixes, an upgrade would mutate such a label.
+is no validation of reserved label prefixes, an upgrade would mutate such a label (this is not a major concern).
+* any clients/applications written to use this label will NOT be backwards compatible with older versions of the apiserver (which DONT enforce this label)
```

Comment:
The worry is actually that they *would* be compatible (i.e., nothing obviously fails), and silently wouldn't have the desired security property.

Reply:
in the conventional scenario, since policies are 'allow only', scheduling would **fail** because the namespace wouldn't have the requisite label, was what I meant, ill clarify that

Reply:
OK thanks.